metrics/density/fast_footprint.sh

```diff
@@ -46,25 +37,9 @@ PARALLELISM="${PARALLELISM:-10}"
 ### The default config - run a small busybox image
 # Define what we will be running (app under test)
 #  Default is we run busybox, as a 'small' workload
-PAYLOAD="${PAYLOAD:-busybox}"
+PAYLOAD="${PAYLOAD:-quay.io/prometheus/busybox:latest}"
 PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
-PAYLOAD_RUNTIME_ARGS="${PAYLOAD_RUNTIME_ARGS:- -m 2G}"
-
-
-#########################
-### Below are a couple of other examples of workload configs:
-#  mysql is a more medium sized workload
-#PAYLOAD="${PAYLOAD:-mysql}"
-# Disable the aio use, or you can only run ~24 containers under runc, as you run out
-# of handles in the kernel. Disable log-bin as it hits a file resize fail error.
-#PAYLOAD_ARGS="${PAYLOAD_ARGS:- --innodb_use_native_aio=0 --disable-log-bin}"
-#PAYLOAD_RUNTIME_ARGS="${PAYLOAD_RUNTIME_ARGS:- -m 4G -e MYSQL_ALLOW_EMPTY_PASSWORD=1}"
-#
-#  elasticsearch is a large workload
-#PAYLOAD="${PAYLOAD:-elasticsearch}"
-#PAYLOAD_ARGS="${PAYLOAD_ARGS:-}"
-#PAYLOAD_RUNTIME_ARGS="${PAYLOAD_RUNTIME_ARGS:- -m 8G}"
-#########################
+PAYLOAD_RUNTIME_ARGS="${PAYLOAD_RUNTIME_ARGS:-5120}"
```

Comment:
should this be `PAYLOAD_RUNTIME_ARGS="${PAYLOAD_RUNTIME_ARGS:- --memory-limit 5120}`?
BTW, why not 2G as before?

Reply:
@chavafg it is not possible to launch containers with ctr with the amount of memory, it takes a lot of time to launch a container which is something that we do not want for this test